hera_cal/omni.py

```diff
@@ -1444,11 +1446,20 @@ def omni_apply(files, opts):
                         np.logical_or(cal.flag_array[antenna_index[ai], nsp, :, :, p1].T,
                                       cal.flag_array[antenna_index[aj], nsp, :, :, p2].T))
 
+        # Define output path and filename
+        inp_filename = f.split('/')[-1]
+        if opts.outpath is None:
+            outpath = '/'.join(f.split('/')[:-1])
+        else:
+            outpath = opts.outpath
+        out_filename = "%s/%s" % (outpath, inp_filename)
```

Comment:
This is more style than substance, but I think we should shy away from this form of string formatting for the more robust `"".format()` statement. So, this could alternatively be written as:
```
out_filename = "{0}/{1}".format(outpath, inp_filename)
```
Not a necessary change, but something to think about.

Reply:
sounds good